test/server/cards/01-Core/RisenFromTheSea.spec.js

```diff
@@ -73,7 +73,7 @@ describe('Risen from the Sea', function() {
 
             it('should not attach the event to the character', function() {
                 expect(this.noAttachmentCharacter.attachments.size()).toBe(0);
-                expect(this.event.location).toBe('being played');
+                expect(this.event.location).toBe('discard pile');
```

Comment:
This should have been an obvious indication something was wrong before this PR. :woman_facepalming: